public/polyfill.js

```diff
@@ -81,5 +81,21 @@
         return getPrototypeChain(Object.getPrototypeOf(y), [..._, y])
     }
 }
+
+// This block fix unimplemented api of Geckoview
+{
+    if (!browser.permissions) {
+        Reflect.defineProperty(browser, 'permissions', {
+            get() {
+                return {
+                    getAll() { return Promise.resolve({}) },
+                    request() { return Promise.resolve() }
```

Comment:
You should save all requested origins (to localstorage, for example) because they will exported in the backup file.